src/services/projects.js

```diff
@@ -496,25 +550,58 @@ module.exports = class ProjectsHelper {
 				}
 
 				if (entityType.validations.regex) {
-					let checkRegex = utils.checkRegexPattern(entityType, fieldData)
-					if (checkRegex) {
-						throw {
-							message: `Special characters not allowed in ${entityType.value}`,
-							error: utils.errorObject(common.BODY, entityType.value),
+					if (entityType.value === common.LEARNING_RESOURCE) {
```

Comment:
@adithyadinesh0412  can we add a generic function to check this learning resource is valid? it repeating in project and task learning resources